requirements.txt

```diff
@@ -0,0 +1,43 @@
+BeautifulSoup
+django-ace == 1.0.4
+django-mptt == 0.9.1
+django-recaptcha == 1.4.0
+djangorestframework == 3.8.2  # Used for pages fixed version as 3.6 causes conflict with collectstatic
+django-signals-ahoy  # Used by satchmo/models.py
+django-redis
+google-api-python-client == 1.7.4
+oauth2client
+html2text == 3.200.3
+htmllaundry
+icalendar == 2.1
+ipython == 5.5.0  # Last version compatible with python 2.7
+Pillow
+pycountry == 1.16
+ephem == 3.7.6.0
+pysftp               # Used for contentserver
+python-dateutil
+python-memcached == 1.59
+python-xmp-toolkit == 1.0.2
+pytz
+requests
+ndg-httpsclient  # Extra for requests
+setuptools
+Wand == 0.4.4
+piexif
+pika
+openpyxl
+django-tinymce4-lite == 1.7.2
+
+# PostgreSQL python client
+psycopg2-binary==2.7.7
+
+# Asynchronous Task Queue
+celery==4.3.0
+
+#############################
+### Testing dependencies ###
+############################
+
+pytest>=4.6.1,<4.7 # 4.6 is the last version to support python 2.7
+pytest-django==3.9.0
+pytest-cov==2.9.0 # coverage report
```

Comment:
Remember Django has its own test suite:

https://docs.djangoproject.com/en/3.0/topics/testing/

Reply:
The existing tests in this project use both the Django test suite and Pytest-django 😁